doc/server/netty.md

```diff
@@ -20,21 +20,49 @@ For example:
 
 ```scala mdoc:compile-only
 import sttp.tapir._
+import sttp.tapir.server.netty.NettyServerType.TCP
 import sttp.tapir.server.netty.{NettyFutureServer, NettyFutureServerBinding}
-
 import scala.concurrent.ExecutionContext.Implicits.global
 import scala.concurrent.Future
+import java.net.InetSocketAddress
 
 val helloWorld = endpoint
   .get
   .in("hello").in(query[String]("name"))
   .out(stringBody)
   .serverLogic(name => Future.successful[Either[Unit, String]](Right(s"Hello, $name!")))
 
-val binding: Future[NettyFutureServerBinding] = 
+val binding: Future[NettyFutureServerBinding[TCP]] = 
   NettyFutureServer().addEndpoint(helloWorld).start()
 ```
 
+## Unix socket support
```

Comment:
I think it's better to use "Domain socket" or IPC, as TCP uses "unix sockets" as well if we are precise :)